python3.8/config.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Env": {
+        "FLASK_APP": "hi.py"
+    },
+    "MapDirs": {
+        "./.venv/*": "/.local",
+        "./usr/lib64/*": "/usr/lib/x86_64-linux-gnu"
```

Comment:
this is fine as an example but it's going to grab everything in that folder which could be massive when in reality you might just be using one or two libs